crates/turbopack-ecmascript/src/transform/mod.rs

```diff
@@ -363,21 +363,6 @@ pub fn remove_shebang(program: &mut Program) {
     }
 }
 
-fn unwrap_module_program(program: &mut Program) -> Program {
-    match program {
-        Program::Module(module) => Program::Module(module.take()),
-        Program::Script(s) => Program::Module(Module {
-            span: s.span,
-            body: s
-                .body
-                .iter()
-                .map(|stmt| ModuleItem::Stmt(stmt.clone()))
-                .collect(),
-            shebang: s.shebang.clone(),
-        }),
```

Comment:
Just noticing that the behavior is not the same. `unwrap_module_program` converts `script` to `module`. We need to keep that behavior

Reply:
Ok, converted to use visitMut instead: 

https://github.com/vercel/turbo/pull/4884/files#diff-b1b0731c098d856a121581436124b8fd726411b872ed875aacce400c825b0fbfR375